views/pages/admin/wishcards.pug

```diff
@@ -4,112 +4,15 @@ block title
-                                                    |
-                                                    = wishItemPrice
-                                            p
-                                                span Item Url:
-                                                    |
-                                                    |
-                                                    a( id='oldWishItemUrl' + card._id href=card.wishItemURL ) Link
-
-                                            input.input1( id='newWishItemUrl' + card._id
-                                                type='text'
-                                                name='newWishItemUrl'
-                                                placeholder='Url for wish item...'
-                                            )
-                                            button.col.wishcard-btn.btn-navy-bg( type='button'
-                                                id='donate-btn-' + card._id data-value-id=card._id
-                                            ) Publish WishCard
-
-append scripts
+block scripts
     script.
         $(document).ready(() => {
-            const unlockCardButtonClick = (e) => {
-                const elementRef = e.target;
-                const wishCardId = elementRef.dataset.valueId;
-                const $oldItemUrl = $('#oldWishItemUrl' + wishCardId);
-                const $newItemUrl = $('#newWishItemUrl' + wishCardId);
-                const wishItemUrl = $newItemUrl.val() !== '' ? $newItemUrl.val() : $oldItemUrl.attr('href');
-
-                console.log(wishItemUrl);
-
-                const toast = new DG.Toast();
-
-                $.ajax({
-                    type: 'PUT',
-                    url: '/admin/wishcards',
-                    data: {
-                        wishCardId,
-                        wishItemUrl,
-                    },
-                    success: (_) => {
-                        toast.show('Card status updated');
-                        removeWishCardFromDOM(elementRef);
-                    },
-                    error: (response) => {
-                        toast.show(response.error.msg, true);
-                    },
-                });
-            }
-
-            const removeWishCardFromDOM = (node) => {
-                const divName = 'row justify-content-center';
-                if (node.parentNode.className !== divName) {
-                    removeWishCardFromDOM(node.parentNode);
-                } else {
-                    while (node.firstChild) {
-                        node.removeChild(node.lastChild);
-                    }
-                }
-            }
-
-            $('.col.wishcard-btn.btn-navy-bg').each(function () {
-                $(this).click(unlockCardButtonClick);
-            })
-        });
\ No newline at end of file
+            const root = ReactRoot(document.querySelector('#wishcard-admin-root'));
```

Comment:
`#wishcard-admin-root` can be replace with `#app-root` defined in the `master.pug` file